src/SeleniumLibrary/keywords/element.py

```diff
@@ -164,21 +164,41 @@ def page_should_contain_element(self, locator, message='', loglevel='INFO'):
         self.assert_page_contains(locator, message=message, loglevel=loglevel)
 
     @keyword
+    def get_matching_locator_count(self, locator):
+        """Returns number of elements matching ``locator``
+
+        If you wish to assert the number of matching elements, use
+        `Locator Should Match X Times`. Keyword will always return an integer.
+
+        Examples assumes that locator matches to three elements
+        | ${count} =      | Get Matching Locator Count | name:div_name  |
+        | Should Be True  | ${count} > 2               |                |
+
+        New in SeleniumLibrary 3.0
+        """
+        return len(self.find_element(locator, first_only=False,
+                                     required=False))
+
+    @keyword
     def locator_should_match_x_times(self, locator, expected_locator_count, message='', loglevel='INFO'):
-        """Verifies that the page contains the given number of elements located by the given `locator`.
+        """Verifies that the page contains the given number of elements located by the given ``locator``.
 
         See `introduction` for details about locating elements.
 
         See `Page Should Contain Element` for explanation about `message` and
         `loglevel` arguments.
+
+        Examples assumes that locator matches to three elements
+        | locator_should_match_x_times | name:div_name | 3 |
```

Comment:
This keyword is so simple that I don't know is an example really needed. If it's kept, the keyword name should be fixed and having just `Example:` before ought to be enough.

Reply:
Done